Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml

```
@@ -932,6 +989,68 @@ script:
     - contextPath: Nexpose.Scan.Status
       description: The scan status. Valid values are aborted, unknown, running, finished, stopped, error, paused, dispatched, integrating
       type: string
+  - name: nexpose-download-report
+    arguments:
+    - name: report_id
+      required: true
+      description: The identifier of the report.
+    - name: instance_id
+      required: true
+      description: The identifier of the report instance. Also supports the "latest"
+        keyword.
+    - name: name
+      description: The report name.
+    - name: format
+      auto: PREDEFINED
+      predefined:
+      - pdf
+      - rtf
+      - xml
+      - html
+      - text
+      description: The report format, default is pdf.
+      defaultValue: pdf
+    outputs:
+    - contextPath: InfoFile.EntryId
+      description: Entry Id of the report file
+      type: string
+    - contextPath: InfoFile.Name
+      description: Name of the report file
+      type: string
+    - contextPath: InfoFile.Extension
+      description: File extension of the report file
+      type: string
+    - contextPath: InfoFile.Info
+      description: Info about the report file
+      type: string
+    - contextPath: InfoFile.Size
+      description: Size of the report file
+      type: number
+    - contextPath: InfoFile.Type
+      description: Type of the report file
+      type: string
+    description: Returns the contents of a generated report.
```

the output is a file, not the content of it.
```suggestion
    description: Returns the generated report.
```